### ISSUE TITLE
SHA256 mismatch for bun upgrade v0.6.9

### ISSUE THREAD

**Issue by LeviticusNelson** [oven-sh/homebrew-bun#7]
```console
==> Fetching oven-sh/bun/bun
==> Downloading https://github.com/oven-sh/bun/releases/download/bun-v0.6.9/bun-darwin-aarch64.zip
Already downloaded: /Users/levi/Library/Caches/Homebrew/downloads/85c974b8cce51d67a36b51e0320d41994aec6db1ea2f292df1d0962245ddb901--bun-darwin-aarch64.zip
Error: SHA256 mismatch
Expected: d24e0f595d80dd13284e24e34ff5faa6a9deb37a922f1815dc07f8abae3f74f6
  Actual: d7d3565ea0ba1cd04fe39218799610404f3f729d0ce554c4b299a0fd27f418c5
    File: /Users/levi/Library/Caches/Homebrew/downloads/85c974b8cce51d67a36b51e0320d41994aec6db1ea2f292df1d0962245ddb901--bun-darwin-aarch64.zip
To retry an incomplete download, remove the file above.
```

**Comment by panaman67** [comment]
I am also seeing this.

**Comment by Electroid** [comment]
Hi, this was because we released 0.5.9 but forgot to bump the minor version in `bun -v`. We're working on a fix.

**Comment by Electroid** [comment]
Fixed by https://github.com/oven-sh/homebrew-bun/commit/62069296cb5913321451d495689e221235691e17
